aviary/mission/flops_based/phases/groundroll_phase.py

```diff
@@ -0,0 +1,202 @@
+        phase.add_timeseries_output(Dynamic.Mission.ALTITUDE)
+        phase.add_timeseries_output("alpha")
+        phase.add_timeseries_output(Dynamic.Mission.FLIGHT_PATH_ANGLE)
+        phase.add_timeseries_output(Dynamic.Mission.THROTTLE)
+
+        return phase
+
+    def make_default_transcription(self):
+        '''
+        Return a transcription object to be used by default in build_phase.
+        '''
+        user_options = self.user_options
+
+        num_segments, _ = user_options.get_item('num_segments')
+        order, _ = user_options.get_item('order')
+
+        seg_ends, _ = dm.utils.lgl.lgl(num_segments + 1)
+
+        transcription = dm.Radau(
+            num_segments=num_segments, order=order, compressed=True,
+            segment_ends=seg_ends)
+
+        return transcription
+
+    def _extra_ode_init_kwargs(self):
+        """
+        Return extra kwargs required for initializing the ODE.
+        """
+        # TODO: support external_subsystems and meta_data in the base class
+        return {
+            'external_subsystems': self.external_subsystems,
+            'meta_data': self.meta_data,
+            'subsystem_options': self.subsystem_options,
+            'set_input_defaults': False,
+        }
+
+
+GroundrollPhase._add_meta_data(
+    'num_segments', val=5, desc='transcription: number of segments')
+
+GroundrollPhase._add_meta_data(
+    'order', val=3,
+    desc='transcription: order of the state transcription; the order of the control'
+    ' transcription is `order - 1`')
+
+GroundrollPhase._add_meta_data('fix_initial', val=False)
+
+GroundrollPhase._add_meta_data('fix_duration', val=False)
+
+GroundrollPhase._add_meta_data('optimize_mach', val=False)
+
+GroundrollPhase._add_meta_data('optimize_altitude', val=False)
+
+GroundrollPhase._add_meta_data('initial_bounds', val=(0., 100.),
+                               units='s', desc='initial bounds')
+GroundrollPhase._add_meta_data('duration_bounds', val=(
+    0., 3600.), units='s', desc='duration bounds')
+GroundrollPhase._add_meta_data('initial_ref', val=100.,
+                               units='s', desc='initial reference')
+GroundrollPhase._add_meta_data('duration_ref', val=1000.,
+                               units='s', desc='duration reference')
+GroundrollPhase._add_meta_data('control_order', val=1, desc='control order')
+GroundrollPhase._add_meta_data('opt', val=True, desc='opt')
+GroundrollPhase._add_meta_data('input_speed_type', val='TAS', desc='input speed type')
+GroundrollPhase._add_meta_data('ground_roll', val=True)
+GroundrollPhase._add_meta_data('rotation', val=False)
+GroundrollPhase._add_meta_data('clean', val=False)
+GroundrollPhase._add_meta_data('balance_throttle', val=False)
+GroundrollPhase._add_meta_data('constraints', val={})
+
+GroundrollPhase._add_initial_guess_meta_data(
+    InitialGuessTime(key='velocity'),
+    desc='initial guess for initial time and duration specified as a tuple')
```

Comment:
should this be velocity, not time?

Reply:
Nope, as we're integrating wrt velocity this is right; I've updated the 'desc' to be more clear